pkg/pipeline/extract/aggregate/aggregate.go

```diff
@@ -55,6 +55,7 @@ type Definition struct {
 
 type GroupState struct {
 	normalizedValues NormalizedValues
+	RecentRawValues  []float64
```

Comment:
Why do we need a list (sorry to be cheap here) .... this will require an unneeded (and uncontrolled) amount of extra memory. Can we have just one value float64 value and add `value` all the time. In the end of the day, this is what you are doing https://github.com/netobserv/flowlogs2metrics/pull/95/files#diff-30d11303a4ad1ec82e4f39e3ae6dc7d044c8dd370c536a98ba90febbfe2d425aR139 ... it is not that we are adding those value with the correct timestamp or something like that ... so why not just add them here?

Reply:
For the case of `Counters`, you are right. We can keep only the sum of the slice, rather than the raw values. But, it won't work for case of `Histograms`. In Histograms, we must use the raw values so they could be put in the right buckets. That's why I chose to use a slice.